fmriprep/interfaces/reports.py

```diff
@@ -114,8 +114,9 @@ def _generate_segment(self):
             t2w_seg = '(+ {:d} T2-weighted)'.format(len(self.inputs.t2w))
 
         # Add list of tasks with number of runs
+        bold_series = self.inputs.bold if isdefined(self.inputs.bold) else []
```

Comment:
No, `InputMultiPath` also considers `[]` to be `Undefined`. And even if it didn't, the fact that the `OutputMultiPath` is pushing out an `Undefined` means we can't expect this to be defined.